chromadb/api/models/CollectionCommon.py

```diff
@@ -161,46 +162,52 @@ def _validate_embedding_set(
         documents: Optional[OneOrMany[Document]],
         images: Optional[OneOrMany[Image]] = None,
         uris: Optional[OneOrMany[URI]] = None,
+    ) -> EmbeddingSet:
+        unpacked_ids = maybe_cast_one_to_many_ids(ids)
+        unpacked_embeddings = maybe_cast_one_to_many_embedding(embeddings)
+        unpacked_metadatas = maybe_cast_one_to_many_metadata(metadatas)
+        unpacked_documents = maybe_cast_one_to_many_document(documents)
+        unpacked_images = maybe_cast_one_to_many_image(images)
+        unpacked_uris = maybe_cast_one_to_many_uri(uris)
+
+        return {
+            "ids": unpacked_ids,
+            "embeddings": unpacked_embeddings,
+            "metadatas": unpacked_metadatas,
+            "documents": unpacked_documents,
+            "images": unpacked_images,
+            "uris": unpacked_uris,
+        }
+
+    def _validate_embedding_set(
```

Comment:
since `_unpack_embedding_set` already returns an `EmbeddingSet`, maybe this should just take an `EmbeddingSet` as an argument?

Reply:
we might still need to assign values of EmbeddingSet into variables to make the code more readable within the function. therefore i believe that this change might not provide that much benefits